teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -30,10 +33,18 @@
       converter = GraffitiConverter.class,
       paramLabel = "<GRAFFITI STRING>",
       description =
-          "Graffiti to include during block creation (gets converted to bytes and padded to Bytes32).",
+          "Graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).",
```

Comment:
There's a stray paren:
```suggestion
          "Graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32.",
```